Sources/xcproj/Deprecations.swift

```diff
@@ -0,0 +1,46 @@
+
+extension PBXProject {
```

Comment:
What do you think about renaming this file to `PBXProject+Deprecations`?

Reply:
Sounds good to me.